src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/repository/ReferralRepository.kt

```diff
@@ -4,4 +4,6 @@ import org.springframework.data.repository.CrudRepository
 import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Referral
 import java.util.UUID
 
-interface ReferralRepository : CrudRepository<Referral, UUID>
+interface ReferralRepository : CrudRepository<Referral, UUID> {
+  fun findByCreatedByUserID(userId: String): List<Referral>
```

Comment:
Can we create an index for this lookup, please? It'll be a major use case to filter the table by user.

Reply:
yep, i'll create a ticket and address is a seperate PR

Reply:
scratch that, i'll just do it now.